db/migrate/20160714083258_update_gas_asset_lists.rb

```diff
@@ -0,0 +1,22 @@
+class UpdateGasAssetLists < ActiveRecord::Migration
+  def change
+    translations = {
+      investment_cost: :initial_investment,
+      lifetime: :technical_lifetime,
+      amount: :units
+    }
+
+    GasAssetList.all.each do |gas_asset|
+      asset_list = gas_asset.asset_list.map do |asset_item|
+        translations.each_pair do |old_key, new_key|
+          asset_item[new_key] = asset_item[old_key]
+          asset_item.delete(old_key)
```

Comment:
FYI: `Hash#delete` returns the deleted value, or `nil` if the given key doesn't exist. So you can do it all in one operation:

```ruby
{ a: 1 }.delete(:a) # => 1
{ a: 1 }.delete(:b) # => nil

new_thing = { a: 1 }.delete(:a) # => 1
```